core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitor.java

```diff
@@ -545,7 +545,8 @@ protected boolean isPeerValid(AdminRow peerRow) {
             && peerRow.getInteger("native_port") != null;
     boolean hasRpcAddress = hasPeersV2RpcAddress || hasPeersRpcAddress;
     boolean hasHostId = peerRow.getUuid("host_id") != null;
-    boolean valid = hasRpcAddress && hasHostId;
+    boolean hasSchemaVersion = peerRow.getUuid("schema_version") != null;
```

Comment:
As I said in the Jira ticket, we should either extend the check to all other columns that might be null and create issues down the line, or chose a better column (datacenter?).
Also, this is not enough to fix the generic problem with duplicate keys in the nodes map.

Reply:
thanks, @adutra! 
I'm not sure if other columns being `null` would qualify the peer row being invalid. For example, `preferred_ip` can be `null` and entry will be still valid. But I agree datacenter feels more relevant. Let me change this for the datacenter. 

>Also, this is not enough to fix the generic problem with duplicate keys in the nodes map. 

when reviewing on `system.peers` row this will be hard. And we would need some check when building that map? because just ignoring duplicate keys could get us into trouble also if we ignore the correct key. 

I would say the driver can get better in ignoring invalid entries where we are certain those are `bogus` entries, and this is already an improvement. If we get two valid entries and we need to ignore one of them we just ignore any?